core/src/main/scala/com/evolutiongaming/kafka/flow/PartitionFlow.scala

```diff
@@ -188,12 +189,13 @@ object PartitionFlow {
 
       clock <- Clock[F].instant
       timestamp <- timestamp updateAndGet (_.copy(clock = clock))
-      states <- cache.values
-      _ <- states.values.toList.parTraverse_ { state =>
-        state flatMap { state =>
-          state.timers.set(timestamp) *>
-            state.timers.trigger(state.flow)
-        }
+
+      keys <- cache.keys
+      states <- keys.toVector.traverseFilter(cache.get)
```

Comment:
could we include all in one `.parTraverse_` and not `.traverseFilter` + `parTraverse_` ?